src/library/Uno.Toolkit.Material/Uno.Toolkit.UI.Material.csproj

```diff
@@ -18,6 +18,21 @@
 		<XamlMergeOutputFile>Generated\mergedpages.uwp.xaml</XamlMergeOutputFile>
 	</PropertyGroup>
 
+	<Choose>
```

Comment:
since we are using this, we can drop the 2files system

Reply:
I'll split that out into another PR with an issue once this is merged